front/lib/api/assistant/configuration.ts

```diff
@@ -482,38 +488,37 @@ async function fetchWorkspaceAgentConfigurationsForView(
       // Retrieval configurations.
       const retrievalActionsConfigurations =
         retrievalActionsConfigurationsPerAgent.get(agent.id) ?? [];
-
       actions.push(...retrievalActionsConfigurations);
 
       // Dust app run configurations.
       const dustAppRunActionsConfigurations =
         dustAppRunActionsConfigurationsPerAgent.get(agent.id) ?? [];
-
       actions.push(...dustAppRunActionsConfigurations);
 
       // Websearch configurations.
       const websearchActionsConfigurations =
         websearchActionsConfigurationsPerAgent.get(agent.id) ?? [];
-
       actions.push(...websearchActionsConfigurations);
 
       // Browse configurations.
       const browseActionsConfigurations =
         browseActionsConfigurationsPerAgent.get(agent.id) ?? [];
-
       actions.push(...browseActionsConfigurations);
 
       // Table query configurations.
       const tableQueryActionsConfigurations =
         tableQueryActionsConfigurationsPerAgent.get(agent.id) ?? [];
-
       actions.push(...tableQueryActionsConfigurations);
 
       // Process configurations.
-      const processActionConfigurations =
+      const processActionsConfigurations =
```

Comment:
I would have made the opposite refactoring (no double plural)

Reply:
Just trying to maintain consistency. Fixed the 1-off insteand of the n-1 :)